File: 10 - Python/Exercícios/04 - Vetores/Exercício 10.py
#Exercício - Aprovados

N = int(input("Quantos alunos serão digitados? "))

nome = [0] * N
nota1 = [0] * N
nota2 = [0] * N

for i in range(N):

    print(f"Digite nome, primeira e segunda nota do {i+1}º aluno:")
    nome[i] = input("")
    nota1[i] = float(input(""))
    nota2[i] = float(input(""))

print("")
print("Alunos aprovados:")

for i in range(N):

    media = (nota1[i] + nota2[i]) / 2

    if media >= 6:
        print(nome[i])

File: 07 - C++/Exercícios/04 - Vetores/Exercício 10.cpp
//Exercício - Aprovados

#include <iostream>
#include <string.h>
#include <climits>

using namespace std;

int main(){

    int i, N;
    double media;

    cout << "Quantos alunos serao digitados? ";
    cin >> N;

    string nome[N];
    double nota1[N], nota2[N];

    for(i = 0; i < N; i++){

        cout << endl << "Digite nome, primeira e segunda nota do " << i+1 << "o aluno:" << endl;
		cin.ignore(INT_MAX, '\n');
        getline(cin, nome[i]);
        cin >> nota1[i];
        cin >> nota2[i];
    }

    cout << endl << "Alunos aprovados:" << endl;

    for(i = 0; i < N; i++){
    
        media = (nota1[i] + nota2[i]) / 2.0;

        if(media >= 6.0){
            cout << nome[i] << endl;
        }
    }
    
    return 0;

}